src/search/components/SearchResultItem.tsx

```diff
@@ -64,14 +65,38 @@ const SearchResultItem: FunctionComponent<SearchResultItemProps> = ({
 			/>
 		);
 	};
+	const checkIfResultIsNotAnItem = (result: Avo.Search.ResultItem): boolean => {
+		return result.administrative_type === ('collectie' || 'bundel' || 'opdracht');
+	};
+
+	const renderAuthorOrOrganisation = (result: Avo.Search.ResultItem) => {
+		if (checkIfResultIsNotAnItem(result)) {
```

Comment:
Prefer to avoid negatives in function names, since they are harder to reason about.
```suggestion
		if (!isItem(result)) {
```

and flip the inner boolean inside the function or change the logic to check for items instead of not items